### ISSUE TITLE
Leaving and reopening the app brings you to previously state/navigated place instead of configured home

### ISSUE THREAD

**Issue by Gelunox** [deckerst/aves#953]
**Describe the bug**
If I exit the app via the back button and then open it again, I'm back to the last root-location I navigated. The configured home-page only opens if I explicitly close the app from the app-switcher.

**To Reproduce**
1. Go to settings
2. Configure `Navigation -> Home` as `All Collection`
3. Go back
4. Navigate to another place, e.g. Albums
5. Press android system back button to leave the app (twice if that setting is on, does not influence bug behaviour)
6. Open app again

**Expected behavior**
After properly exiting and reopening the app I expect to be back on home, if I leave the app by switching apps without exiting the app first, I expect it to stay on the last opened location.

**Screenshots**
assuming n/a

**System information and logs:**
Package: deckers.thibault.aves
Installer: com.android.vending
Aves version: 1.10.7-play, build 116
Flutter: stable 3.19.3
Android version: 13, API 33
Android build: lineage_polaris-userdebug 13 TQ3A.230901.001 adbe221247
Device: Xiaomi Mi MIX 2S
Support: dynamic colors=true, geocoder=true, HDR=true
Mobile services: ready
System locales: en_GB
Storage volumes: /storage/emulated/0/
Storage grants: 
Error reporting: true

**Additional context**
This could also be interpreted as a feature-request for a configuration option to do this, but in my opinion either way is fine.

<sub>Thank you for your efforts in creating this app.</sub>

**Comment by deckerst** [comment]
It's funny because a few days ago, someone complained of the exact opposite behaviour (reopening on home instead of the last page), and they were also on lineage...

I believe the current behaviour mostly depends on how/when the OS kills the app, and how you reopen it: by tapping on the launcher icon (which probably reopens on home), or via the recent apps switcher (which probably reopens on last page, if the app wasn't killed in between).

**Comment by Gelunox** [comment]
If I back out normally the app isn't killed immediately, I'm assuming it has to be in the background for at least a few minutes before the system kills it.

It has been many years since I touched android code, but I assume there's a not-too-difficult way to tell the intent it should be cleared when exiting the app, and make it configurable if that behaviour should happen.

Truthfully I'm just extremely lazy, I disabled the bottom navigation bar because I don't like it, and the amount of effort I have to put in to move my hand to the top left and click the hamburger menu and then click all collection is much higher than just pressing (or swiping, I use swipe navigation) back a few times and opening the app again from my launcher.
